spec/api/holdings_api_spec.rb

```diff
@@ -232,8 +232,62 @@ def parse_body(call)
         build(:holding, organization: "umich", ocn: ocn),
         build(:holding, organization: "smu", ocn: ocn)
       )
-      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1, organization: "umich")))
+      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1)))
       expect(response["organizations"].sort).to eq ["smu", "umich"]
     end
   end
+
```

Comment:
For these tests at least we don't need any of the organizations to be real organizations, so I went with a..d.